### PR TITLE
PR2 exact species assignment now without taxa ending with sp.

### PR DESCRIPTION
**Pull request by d4straub**
Closes https://github.com/nf-core/ampliseq/issues/593.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/taxref_reformat_pr2.sh
Comment (jtangrot) [inline bin/taxref_reformat_pr2.sh:1]: I suggest to add a comment on the last part too, i.e. the awk part that removes " sp".

**Review by jtangrot — APPROVED** [review]
LGTM!

File: bin/taxref_reformat_pr2.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ gunzip -c *dada2.fasta.gz > assignTaxonomy.fna
 
 # For addSpecies(), the UTAX file is downloaded and reformated to only contain the id and species.
 # The second two sed calls are to replace "_" with space only in the species name and not the last part of the id (overdoing it a bit, as I don't the id actually matters as long as it's unique).
-gunzip -c *UTAX.fasta.gz | sed '/^>/s/>\([^;]*\);.*,s:\(.*\)/>\1 \2/' | sed 's/_/ /g' | sed 's/ \([A-Z]\) /_\1 /' > addSpecies.fna
+gunzip -c *UTAX.fasta.gz | sed '/^>/s/>\([^;]*\);.*,s:\(.*\)/>\1 \2/' | sed 's/_/ /g' | sed 's/ \([A-Z]\) /_\1 /' | awk '!/ sp.\n/' RS=">" ORS=">" > addSpecies.fna
```
Comment (d4straub) [inline bin/taxref_reformat_pr2.sh:10]:
```suggestion
# The awk part removes any entries (sequence name and sequence) that have a sequence name ending with " sp."
gunzip -c *UTAX.fasta.gz | sed '/^>/s/>\([^;]*\);.*,s:\(.*\)/>\1 \2/' | sed 's/_/ /g' | sed 's/ \([A-Z]\) /_\1 /' | awk '!/ sp.\n/' RS=">" ORS=">" > addSpecies.fna
```

File: bin/taxref_reformat_pr2.sh
Comment (d4straub) [reply bin/taxref_reformat_pr2.sh:1]: Thanks, yes, will do!

File: bin/taxref_reformat_pr2.sh (hunk 1 above)
Comment (jtangrot) [reply bin/taxref_reformat_pr2.sh:11]: Looks good, thanks!

**Pull request merged**